docs: Community Spotlight - ATrueLight4 (#2168)
block/goose: documentation/blog/2025-04-14-community-atruelight4/index.md

---
title: "How ATrueLight4 Helped Goose Navigate Windows"

> description: "description here"

Reviewer: ```suggestion
description: "Block Open Source Top Contributor of the Month"
```

feel free to dismiss if you have something better

authors: 
    - tania
---

![blog cover](goose4win.png)

As the goose team continues to work on new features over 800 community members voted to express their need for Goose on Windows. Today's top contributor stepped in to aid this ongoing effort.

<!--truncate-->

## Windows Support

Since the launch of codename goose, the open source community has been wanting and requesting the same automation by goose on Windows.

Currently, goose's CLI version works on Windows with limited built-in extensions via WSL, or Windows Subsystem for Linux. While there is a limited internal beta in the works, there are several bugs and improvements that are being worked on to ensure codename goose has the same seamless experience as it does on Mac.

## ATrueLight4's Goose4Win

> Our featured contributor, ATrueLight4, contributed Goose4Win, an effort that improves Windows-specific path handling. Their contribution was merged into main, which means if you download goose for CLI on Windows right now? You don’t need to install anything else to enjoy this change.

Reviewer: ```suggestion
Our featured contributor, ATrueLight4, contributed Goose4Win, an effort that improves Windows-specific path handling. Their contribution was merged into main, which means if you download goose for CLI on Windows right now, you don’t need to install anything else to enjoy this change.
```

> Our featured contributor, ATrueLight4, contributed Goose4Win, an effort that improves Windows-specific path handling. Their contribution was merged into main, which means if you download goose for CLI on Windows right now? You don’t need to install anything else to enjoy this change.

Reviewer: ```suggestion
Our featured contributor, [ATrueLight4](https://github.com/ATrueLight4), contributed Goose4Win, an effort that improves Windows-specific path handling. Their contribution was merged into main, which means if you download goose for CLI on Windows right now? You don’t need to install anything else to enjoy this change.
```

> Now when you run goose CLI on Windows, you won’t run into path navigation issues, like when you try to get Goose to look at specific folders. Thank you so much for your contribution, ATrueLight4! Your work brings us one step closer to Goose on Windows.

Reviewer: ```suggestion
Thank you so much for your contribution, ATrueLight4! Your work brings us one step closer to Goose on Windows.
```

> ## As A Thank You

Reviewer: ```suggestion

```

> To show our deep gratitude for your contribution and time, please enjoy your Top Contributor badge on the Block Open Source Discord! You also get to be one of the first top contributors to receive exclusive codename goose swag. (more info on that to be announced later 👀🪿)

Reviewer: ```suggestion
To show our deep gratitude for your contribution, we've granted you the exclusive ✨Top Contributor✨ badge on the Block Open Source Discord! You'll also be one of the first contributors to receive exclusive codename goose swag. (more info on that to be announced later 👀🪿)
```

## Get Your Own Spotlight

> Interested in contributing to Goose, and getting your contribution featured? Whether it's improving platform support, adding new features, or fixing bugs, we welcome all contributions from the open source community. You can [join the Block Open Source Discord](https://discord.gg/block-opensource) or [get started using codename goose](https://block.github.io/goose/) today.

Reviewer: ```suggestion
Interested in contributing to goose and having your contribution featured? Whether it's improving platform support, adding new features, or fixing bugs, we welcome all contributions from the open source community. You can [join the Block Open Source Discord](https://discord.gg/block-opensource) or [get started using codename goose](https://block.github.io/goose/) today.
```

<head>
  <meta property="og:title" content="How ATrueLight4 Helped Goose Navigate Windows" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="https://block.github.io/goose/blog/2025/04/14/community-atruelight4" />

>   <meta property="og:description" content="description here" />

Reviewer: ```suggestion
  <meta property="og:description" content="Block Open Source Top Contributor of the Month" />
```

  <meta property="og:image" content="https://block.github.io/goose/assets/images/goose4win-7f57433fa3f19849e74b18ebffe08bcf.png" />
  <meta name="twitter:card" content="summary_large_image" />
  <meta property="twitter:domain" content="block.github.io/goose" />
  <meta name="twitter:title" content="How ATrueLight4 Helped Goose Navigate Windows" />

>   <meta name="twitter:description" content="description here" />

Reviewer: ```suggestion
  <meta name="twitter:description" content="Block Open Source Top Contributor of the Month" />
```
